src/wallet/rpcdump.cpp

```diff
@@ -217,6 +221,29 @@ Value importaddress(const Array& params, bool fHelp)
     return Value::null;
 }
 
+Value z_importwallet(const Array& params, bool fHelp)
+{
+    if (!EnsureWalletIsAvailable(fHelp))
+        return Value::null;
+
+    if (fHelp || params.size() != 1)
+        throw runtime_error(
+            "z_importwallet \"filename\"\n"
```

Comment:
I probably should have raised this earlier, but: do we really need to be able to import/export wallet files that exclude the zkeys? I bet that RPC-applications ported from Bitcoin are more likely to work as intended if zkeys were implicitly included: they *might* break if they make certain assumptions regarding the keys, but they *will* violate user expectations if the zkeys are just dropped.